internal/models/streamup.go

```diff
@@ -15,3 +15,21 @@ type StreamUpEventSubEvent struct {
 	Type                 string `json:"type"`
 	StartedAt            string `json:"started_at"`
 }
+
+type StreamUpWebSubResponse struct {
+	Data []StreamUpWebSubResponseData `json:"data"`
+}
+
+type StreamUpWebSubResponseData struct {
+	ID    			string  	`json:"id"`
+	UserID  		string  	`json:"user_id"`
+	UserName  		string  	`json:"user_name"`
+	GameID  		string  	`json:"game_id"`
+	CommunityIDs	[]string  	`json:"community_ids"`
+	Type    		string 		`json:"type"`
+	Title    		string 		`json:"title"`
+	ViewerCount     int64       `json:"viewer_count"`
+	StartedAt       string 		`json:"started_at"`
+	Language 		string 		`json:language`
+	ThumbnailURL 	string 		`json:thumbnail_url`
+}
```

Comment:
The docs are a little wrong here; https://dev.twitch.tv/docs/api/reference#get-streams is correct and not the webhooks reference.